src/components/headers_overlay/headers_overlay.xml

```diff
@@ -42,25 +42,22 @@
         t-as="hiddenItem"
         t-key="hiddenItem_index">
         <div
-          class="o-unhide-buttons position-relative float-end"
+          class="position-absolute end-0 translate-middle-y"
           t-att-style="getUnhideButtonStyle(hiddenItem[0])">
-          <t t-if="!hiddenItem.includes(0)">
-            <div
-              class="o-unhide rounded mb-1"
-              t-att-data-index="hiddenItem_index"
-              t-on-click="() => this.unhide(hiddenItem)">
-              <t t-call="o-spreadsheet-Icon.TRIANGLE_UP"/>
-            </div>
-          </t>
-          <t
-            t-if="!hiddenItem.includes(env.model.getters.getNumberRows(env.model.getters.getActiveSheetId())-1)">
-            <div
-              class="o-unhide rounded"
-              t-att-data-index="hiddenItem_index"
-              t-on-click="() => this.unhide(hiddenItem)">
-              <t t-call="o-spreadsheet-Icon.TRIANGLE_DOWN"/>
-            </div>
-          </t>
+          <div
+            class="o-unhide rounded mb-1"
+            t-att-class="{'invisible': hiddenItem.includes(0)}"
```

Comment:
Any reason to change the `t-if` to `invisible` class ? Does it break the style with `t-if` ?
It's best to use `t-if` to avoid building the DOM (to make it invisible afterward)

Reply:
> Any reason to change the `t-if` to `invisible` class ? Does it break the style with `t-if` ?

Yes, there is. If the icon is positioned on the first or last row/column, it won't have one of the icons, leading to a misalignment.

I attempted to resolve this by adding some pixels, but then ADRM proposed using the invisible class instead. You can find the discussion [here](https://github.com/odoo/o-spreadsheet/pull/3733#discussion_r1497591212).